docs/tutorials/TOPP/data-analysis-in-toppview.md

```diff
@@ -0,0 +1,38 @@
+Data Analysis in TOPPView
+=========================
+
+TOPPView also offers limited data analysis capabilities for single layers, which will be illustrated in the following
```

Comment:
What do you mean by "single layers"?

Reply:
Hmm we have the concept of data layers similar to layers in photoshop. But this might not be well explained by us

Reply:
https://github.com/OpenMS/OpenMS-docs/issues/76.